Project-Hotpot/Project-Hotpot/Managers/SpotifyManager.swift

```diff
@@ -70,6 +72,7 @@ class SpotifyManager: NSObject {
     // MARK: - Initializers
     //doing this way so that only 1 instance can be created
     override private init() {
+        self.genreSeedArray = []
```

Comment:
Few questions about this approach:
- What happens when user opens the app and there is no internet. Suddenly network comes back, now user wants to search. What do you do?
- If fetching on every app load efficient? How often do these genres change in reality? Is there a more efficient way to fetch this.